testing/ffcall/APKBUILD

```diff
@@ -36,10 +36,7 @@ check() {
 
 package() {
 	cd "$builddir"
-	mkdir -p "$pkgdir"/usr/share/man \
```

Comment:
why this line removed?

Reply:
`make install` already creates the required directory structure `/usr/{lib,include,share}` in the destination directory. The line above simply creates the same dir again.